### ISSUE TITLE
Add shortcut to emojis in SMS apps

### ISSUE THREAD

**Issue by keeganjk** [klausw/hackerskeyboard#604]
I use this keyboard for everything, of coding, text-editing, and terminals. In the SMS app, it has a shortcut to :-) and 14 other similar ASCII-Art emojis. However, these aren't always enogh for texting. Could you add a shortcut to the emojis I would find in GBoard (in SMS app)?

**Comment by FRLanc** [comment]
Yes, please, same thing for instagram and other. It would be nice to be able to turn emojis button in config for some apks may be.

**Comment by TPS** [comment]
Duplicates #376
